cleanlab/count.py

```diff
@@ -114,6 +116,13 @@ def num_label_issues(
       The estimated number of examples with label issues in the dataset.
     """
     valid_methods = ["off_diagonal", "off_diagonal_calibrated"]
+    if isinstance(confident_joint, np.ndarray):
+        warn_str = (
+            "WARNING! The supplied 'confident_joint' is ignored. Instead the 'confident_joint' is "
+            "estimated from the passed in 'labels' and 'pred_probs'."
+        )
```

Comment:
```suggestion
            "The supplied `confident_joint` is ignored. Instead the `confident_joint` is "
            "re-estimated from the provided `labels` and `pred_probs`."
        )
```

Reply:
as I mentioned above, would be nice to offer another `estimation_method` option in `num_label_issues()` which does use the `confident_joint` if provided by user. So that way this warning message can point the user to that `estimation_method` option